server/src/completion-provider/ember-helpers.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  CompletionItem,
+  CompletionItemKind
+} from 'vscode-languageserver';
+
+type UsableIn = 'BlockPath' | 'MustachePath' | 'SubExpressionPath';
+type EmberHelperConfig = [string, CompletionItemKind, UsableIn[]];
```

Comment:
I'd prefer it if we could make this a proper class with a constructor and/or factory function.